### DISCUSSION TITLE
How to get option BUILD_CUDA_MODULE=ON in my project

### DISCUSSION THREAD

**Discussion by godfather1402** [isl-org/Open3D#3812] [Q&A]
Hi,
I would like to use some utility functions of Open3D in my project with the flag `-DBUILD_CUDA_MODULE=ON` , i.e: 
`https://github.com/isl-org/Open3D/blob/master/cpp/open3d/core/CUDAUtils.h`
`https://github.com/isl-org/Open3D/blob/master/cpp/open3d/core/MemoryManager.h`  and so on. 
But somehow the flag `BUILD_CUDA_MODULE` is OFF if I use Open3D library in my project, although I built Open3D from source with the flag `-DBUILD_CUDA_MODULE=ON`.
Open3D is found in My CmakeLists.txt with commands:
```
find_package(Open3D HINTS ${CMAKE_INSTALL_PREFIX}/lib/CMake)
list(APPEND Open3D_LIBRARIES dl)
set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} ${Open3D_C_FLAGS}")
set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} ${Open3D_CXX_FLAGS}")
set(CMAKE_EXE_LINKER_FLAGS "${CMAKE_EXE_LINKER_FLAGS} ${Open3D_EXE_LINKER_FLAGS}")
link_directories(${Open3D_LIBRARY_DIRS})

include_directories(
    include
    ${Open3D_INCLUDE_DIRS}
    )
target_link_libraries(${PROJECT_NAME}
    ${Open3D_LIBRARIES}
    )
```
Do you know how can I use source code in Open3D with the flag  `-DBUILD_CUDA_MODULE=ON`.
Thank you

**Comment by ssheorey** [comment]
Hi @godfather1402 if cmake is not able to find your CUDA libraries, it will turn BUILD_CUDA_MODULE to OFF for the build. Check your cmake build logs to see if that happens. See the CMake FindCUDAToolkit documentation to find out how to make sure CMake can find CUDA:
https://cmake.org/cmake/help/latest/module/FindCUDAToolkit.html

Also, you can now just download prebuilt Open3D binaries (with CUDA) from GIthub releases.
